includes/autoloader.php

```diff
@@ -92,6 +92,7 @@ private static function init_classes_map() {
 			'Conditions' => 'includes/conditions.php',
 			'Controls_Manager' => 'includes/managers/controls.php',
 			'Controls_Stack' => 'includes/base/controls-stack.php',
+			'Sub_Controls_Stack' => 'includes/base/sub-controls-stack.php',
```

Comment:
Our goal is to get rid of this list and automatically load everything by NS. Therefore, this class shouldn't be on this list and has to be loaded from 'core/base' or something like this

Reply:
It should be near the control stack itself.